react/components/SelectionListOrderBy.js

```diff
@@ -105,7 +106,7 @@ const SelectionListOrderBy = ({
           >
             {sortByMessage}
           </span>{' '}
-          {getOptionTitle(orderBy)}
+          {showOrderTitle === true ? getOptionTitle(orderBy) : ''}
```

Comment:
```suggestion
          {showOrderTitle ? getOptionTitle(orderBy) : null}
```

Reply:
Done :) I tried in that way in the first place and it didn't work.. that's why i put `=== true`